packages/@uppy/dashboard/src/components/FileCard/index.js

```diff
@@ -113,9 +113,9 @@ class FileCard extends Component {
             className="uppy-DashboardContent-back"
             type="button"
             title={this.props.i18n('finishEditingFile')}
-            onClick={this.handleSave}
+            onClick={this.handleCancel}
           >
-            {this.props.i18n('done')}
+            {this.props.i18n('cancel')}
```

Comment:
I don't understand this, there's no longer a `Save` button?

Reply:
So, “Done” was supposed to cancel, there are Save buttons on Image Editor and Meta Editor screens.